tests/model/test_integrands.py

```diff
@@ -18,7 +18,7 @@ def test_make_average_integrand_cases_from_gbd(mocker):
 
     ec = make_execution_context(location_id=180)
 
-    average_integrand_cases = make_average_integrand_cases_from_gbd(ec)
+    average_integrand_cases = make_average_integrand_cases_from_gbd(ec, [1, 2])
```

Comment:
If this were my PR, would someone like MikeR call these mystery numbers?

Reply:
Maybe. Those are such universally used magic number here that I can hardly see them any more.